src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -1081,7 +1081,7 @@ internal Tool CreateTool(ToolVersionOne v1Tool)
                 {
                     DottedQuadFileVersion = v1Tool.FileVersion,
                     FullName = v1Tool.FullName,
-                    Language = v1Tool.Language,
+                    Language = v1Tool.Language ?? "en-US",
```

Comment:
>"en-US" [](start = 50, length = 7)

This value (and other terms) could usefully be stored as const values somewhere. I propose that jschema create (and itself refer to) these.

[https://github.com/Microsoft/jschema/issues/90](https://github.com/Microsoft/jschema/issues/90)